modules/administration/pages/monitoring.adoc

```diff
@@ -145,136 +76,150 @@ zypper in golang-github-prometheus-prometheus
-The packages are available from the {productname} client tools channels, and can be enabled and configured directly in the {productname} {webui}.
-Currently, two exporters are supported:
+.Procedure: Installing Grafana via {productname} {webui}
 
-* Node exporter: [systemitem]``golang-github-prometheus-node_exporter``.
-See https://github.com/prometheus/node_exporter.
-* PostgreSQL exporter: [systemitem]``golang-github-wrouesnel-postgres_exporter``.
-See https://github.com/wrouesnel/postgres_exporter.
-
-Installing and configuring exporters is done using a Salt formula.
+. In the {productname} {webui}, open the details page of the system where Grafana is to be installed, and navigate to the [guimenu]``Formulas`` tab.
+. Check the [guimenu]``Grafana`` checkbox to enable  monitoring formulas, and click btn:[Save].
+. Navigate to the ``Grafana`` tab in the top menu.
+. In the ``Enable and configure Grafana`` section, enter the admin credentials you want to use to log in Grafana.
+. On the ``Datasources`` section, make sure that the Prometheus URL field points to the system where Prometheus is running.
+. Customize any other configuration options according to your needs. 
+. Click btn:[Save Formula].
+. Apply the highstate.
+. Once the highstate completes, check that the Grafana interface is loading correctly. In your browser, navigate to the URL of the server where Grafana is installed, on port 3000 (for example, [literal]``http://example.com:3000``).
 
-When you have the exporters installed and configured, you can begin using Prometheus to scrape metrics from monitored systems.
-You can do this directly through the {productname} {webui}, or set up service discovery.
-Service discovery instructs Prometheus to automatically scrape metrics from systems as they are enabled.
+[NOTE]
+{productname} provides pre-build dashboards for Server self-health, basic client monitoring, and more. In the formula configuration page, you can select which dashboards to provision in your Grafana setup.
 
+.Procedure: Manually installing Grafana
 
+. Install the [package]``grafana`` package:
++
+----
+zypper in grafana
+----
+. Enable the Grafana service:
++
+----
+systemctl enable --now grafana-server
+----
+. Check that the Grafana interface is loading correctly.
+In your browser, navigate to the URL of the server where Grafana is installed, on port 3000 (for example, [literal]``http://example.com:3000``).
 
-== Enable and Configure Monitoring
+image::monitoring_grafana_example.png[scaledwidth=80%]
 
+[NOTE]
+For more information on how to manually install and configure Grafana, see https://grafana.com/docs
 
-.Procedure: Enabling Self Monitoring for {productname}
 
-. In the {productname} {webui}, navigate to menu:Admin[Manager Configuration > Monitoring].
-. Click btn:[Enable services].
+== Configure {productname} Monitoring
 
-image::monitoring_enable_services.png[scaledwidth=80%]
+=== Server Self Monitoring
```

Comment:
Probably don't need this heading.

Reply:
Server self monitoring and monitoring client systems are independent topics. How to distinguish them otherwise?

Reply:
You can leave them in if you put a sentence or two between them.

Reply:
Sentence added